core/src/lib.rs

```diff
@@ -298,6 +298,7 @@ impl Schema {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
 
+    /// Returns an edn::Value::Vector representation of the schema map.
```

Comment:
nit: `edn::value::Vector` in backticks.  (This will get formatted by Rustdoc eventually; it's some flavour of Markdown.)

nit: this uses the more than `schema_map`, so let's say "of the schema".  I think it's work saying that it returns a "symbolic representation", too; and maybe even "symbolic representation suitable for applying across Mentat stores".